.env.example

```diff
@@ -1,28 +1,39 @@
 # PUBLIC ENV VARS
 
 # General
-NEXT_PUBLIC_DAO_ADDRESS=0xd9A924bF3FaE756417b9B9DCC94C24681534b8F7
+NEXT_PUBLIC_DAO_ADDRESS="0x9B92805eE0ABF2DdFC8E0d684c6524D1Bf6ea81C"
 NEXT_PUBLIC_TOKEN_ADDRESS=0x9A3218197C77F54BB2510FBBcc7Da3A4D2bE0DeE
 
-# Plugin addresses
+# MACI voting
+NEXT_PUBLIC_MACI_VOTING_PLUGIN_ADDRESS="0x316516EdA7eacff4EA91fec99742d5D4360414e9"
+NEXT_PUBLIC_MACI_ADDRESS="0x0523879d6697BDe8AEe5b3abAc9C4193bDdd6882"
+NEXT_PUBLIC_MACI_DEPLOYMENT_BLOCK="0"
+NEXT_PUBLIC_SECONDS_PER_BLOCK="1"
```

Comment:
What blocks are these? If it's Arbitrum, Does that conflict with using ETH Sepolia?

Reply:
`NEXT_PUBLIC_MACI_DEPLOYMENT_BLOCK` is when the MACI contract was deployed in order to avoid looking for all the blockchain logs and only look from this specific block onwards 😊 


`NEXT_PUBLIC_SECONDS_PER_BLOCK` was used in the future function `getBlockFromFutureTimestamp` (trying to fix the Arbitrum L1-L2 issues)

Reply:
I think we can leave it here until we implement it on Arbitrum :)